src/ContinuumAbsorption/absorption_ff_neutral_metals_molecules.jl

```diff
@@ -0,0 +1,422 @@
+#                              1708.0  122.0   29.2   NaN  NaN] .* 1e-26)
+#
+#    _build_interpolator_john75b(λ_vals, T_vals, elements)
+#end
+
+
+# -----------------------------------------------------------------------------------------
+# Define absorption functions using data from papers where Bell & Berrington were coauthors
+# -----------------------------------------------------------------------------------------
+
+"""
+    _bell_berrington_negative_ion_ff_absorption(ν, T, ndens_neutral, ne, cross_section_interp)
+    
+Each paper presenting absorption coefficient data that Bell & Berrington are co-authors on all 
+seem to present their data in a similar format
+"""
+function _bell_berrington_ff_neg_ion_absorption(ν, T, ndens_neutral, ne, cross_section_interp)
+    # cross_section_interp should include the correction for stimulated emission
+    θ = 5040.0 / T
+    λ = c_cgs * 1.0e8 / ν # in ångstroms
+    Pₑ = ne * kboltz_cgs * T # partial pressure contributed by electrons
+    ndens_neutral * Pₑ * cross_section_interp(λ, θ)
+end
+
+const _Cminus_ff_coef = let
+    # this data was transcribed from table 3 of Bell+ 1988
+    # (https://ui.adsabs.harvard.edu/abs/1988JPhB...21.2319B)
+    #
+    # these cross-sections include the stimulated emission correction
+
+    λ_vals = SA[151890.0, 113918, 91134, 45567, 30378, 22784, 18227, 15189, 13019, 11392, 10126,
+                9113, 7595, 6510, 5696, 5063, 4557, 3645, 3038]
+    θ_vals = SA[0.5, 0.6, 0.8, 1.0, 1.2, 1.6, 2.0, 2.8]
+
+    entries =   [ 3.95e2 4.62e2 6.13e2 7.83e2 9.68e2 1.35e3 1.73e3 2.37e3;
+                  2.23e2 2.61e2 3.47e2 4.43e2 5.48e2 7.67e2 9.78e2 1.34e3;
+                  1.43e2 1.68e2 2.23e2 2.86e2 3.53e2 4.94e2 6.29e2 8.57e2;
+                  3.72e1 4.40e1 5.93e1 7.66e1 9.51e1 1.33e2 1.69e2 2.31e2;
+                  1.74e1 2.08e1 2.83e1 3.67e1 4.54e1 6.22e1 7.71e1 1.00e2;
+                  1.04e1 1.26e1 1.75e1 2.28e1 2.81e1 3.79e1 4.61e1 5.74e1;
+                  7.20   8.80   1.24e1 1.61e1 1.99e1 2.66e1 3.19e1 3.88e1;
+                  5.41   6.69   9.50   1.25e1 1.54e1 2.06e1 2.47e1 3.04e1;
+                  4.32   5.39   7.73   1.02e1 1.25e1 1.68e1 2.02e1 2.52e1;
+                  3.60   4.52   6.54   8.61   1.06e1 1.42e1 1.71e1 2.15e1;
+                  3.12   3.96   5.80   7.69   9.56   1.27e1 1.52e1 1.88e1;
+                  2.79   3.57   5.28   7.05   8.80   1.15e1 1.37e1 1.70e1;
+                  2.29   2.97   4.40   5.87   7.27   9.77   1.18e1 1.45e1;
+                  1.98   2.57   3.85   5.15   6.42   8.70   1.06e1 1.31e1;
+                  1.75   2.29   3.45   4.63   5.79   7.88   9.62   1.20e1;
+                  1.58   2.08   3.14   4.22   5.29   7.25   8.88   1.12e1;
+                  1.45   1.91   2.90   3.92   4.92   6.77   8.31   1.05e1;
+                  1.22   1.62   2.49   3.39   4.29   5.95   7.35   9.33;
+                  1.07   1.43   2.21   3.04   3.86   5.38   6.67   8.50] .* 1e-27
```

Comment:
No need to change it, but I don't think you need the semicolons here.

Reply:
The rules for array literals always confuse me